doc/ENCRYPTION.md

```diff
@@ -0,0 +1,115 @@
+## Overview
+AES Cipher block chaining (CBC) mode and AES Counter (CTR) mode are supported in pgmoneta. The default setup is no encryption. 
+
+CBC is the most commonly used and considered save mode. Its main drawbacks are that encryption is sequential (decryption can be parallelized).
+
+Along with CBC, CTR mode is one of two block cipher modes recommended by Niels Ferguson and Bruce Schneier. Both encryption and decryption are parallelizable.
+
+Longer the key length, safer the encryption. However, with 20% (192 bit) and 40 (256 bit) extra workload compare to 128 bit.
+
```

Comment:
I would list the available options here as well, and highlight that `aes` is `aes-256-cbc`